src/app.js

```diff
@@ -22,52 +25,103 @@ export default (options = {}) => {
   options = {
     el: '#app',
     name: 'compose',
-    template: '<router-view v-if="loaded"/>',
+    template: `<div>
```

Comment:
This is a bit too much code in the template for my taste. 
Should be refactored into a view.